src/urllib3/contrib/socks.py

```diff
@@ -51,7 +51,7 @@
         (
             "SOCKS support in urllib3 requires the installation of optional "
             "dependencies: specifically, PySocks.  For more information, see "
-            "https://urllib3.readthedocs.io/en/latest/contrib.html#socks-proxies"
+            "https://urllib3.readthedocs.io/en/stable/advanced-usage.html#socks-proxies"
```

Comment:
I had missed from my phone that contrib was changed to advanced-usage, sorry. But this is the main branch, so we should stick with `latest` please:

```suggestion
            "https://urllib3.readthedocs.io/en/latest/advanced-usage.html#socks-proxies"
```